apps/client/src/common/api/settings.ts

```diff
@@ -19,3 +19,19 @@ export async function getSettings(): Promise<Settings> {
 export async function postSettings(data: Settings): Promise<AxiosResponse<Settings>> {
   return axios.post(settingsPath, data);
 }
+
+/**
+ * HTTP request to mutate show welcome dialog
+ * if no value is supplide will just return the current value
+ */
+export async function getShowWelcomeDialog(): Promise<AxiosResponse<{ show: boolean }>> {
+  return axios.get(`${settingsPath}/welcomedialog`);
+}
```

Comment:
question: who is using this endpoint? if there are no current uses we should remove it